systemd/system-generators/coreos-autologin-generator

```diff
@@ -52,6 +64,8 @@ autologin() {
         overlay_unit "getty@.service"
         overlay_unit "serial-getty@.service"
     fi
+
+    emergency_autologin
```

Comment:
last one, so can be simplified to merely: ```grep -q "$1$" /sys/class/tty/console/active || return``` if "$1" is blank it'll still match so no need for the extra if.